tests/main/snap-quota-journal/test-snapd-journal-quota/bin/logger

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Services associated with a specific log namespace may log via syslog, 
+# the native logging protocol of the journal and via stdout/stderr; the logging 
```

Comment:
This talks about syslog, native journal logging and stdout/stderr but the code only does "echo". Why are the others mentioned here? maybe the comment could be tweaked a bit to make this clearer?

Reply:
Yea I guess it doesn't need to be in the file, I was just adding it as an informational piece as to what we can use to log in namespaces with. Unfortunately 'logger' logs directly to /dev/log which you can't when in a namespace. Echo logs directly to stdout, which you can.